TransferMethod/Sources/Coordinators/TransferMethodCoordinators.swift

```diff
@@ -113,7 +114,7 @@ public class AddTransferMethodCoordinator: NSObject, HyperwalletCoordinator {
         self.parentController = parentController
     }
 }
-
+/// ListTransferMethodsCoordinator
```

Comment:
Update to - /// Coordinator class for ListTransferMethods

Reply:
Updated